test/scenario-test.ts

```diff
@@ -26,6 +26,7 @@ describe('ScenarioTest', () => {
         ['boolean', [TestMode.ExtDefault]],
         ['datafactory', [TestMode.ExtDefault]],
         ['managed-network', [TestMode.ExtDefault]],
+        ['msgraphuser', [TestMode.ExtDefault]],
```

Comment:
Is it possible to cover your change in the unit test level?

Reply:
I don't think it's a good idea, 
In msgraph's case, we not only test the merger plugin but also the parameter construction logic in custom.py.

the merger is a very complex logic plugin,  Merge two code model is not naturally supported by autorest, the reason why we can make merge two code models work is because we have overwritten the autorest python pipeline  and autorest/clicommon, The cost of mocking the merge code model condition to test it as unittest would be very high.

The custom.py has not changed into template based yet, it's would be costly to stimulate all the data and constuction logic etc.

Other reasons why I add msgraph into our scenario is that：
1. it's open api 3.0 https://github.com/Azure/autorest.az/pull/761/files#diff-17856715eae6afbf40e9e4c040d033962d42bb4bdb30615c8288fd490d22081b, This is our first time to add openapi 3.0 scenario test case. 
2. Our current command split and merge logic is not correct on this msgraph user https://github.com/Azure/autorest.az/pull/761/files#diff-c95083af46065bc635d1536aaca2bc2bc7d9002264822c715391154f6d0ad290R617-R645 
3. Our current multi-method command body parameter construction logic is also incorrect on this msgraph user https://github.com/Azure/autorest.az/pull/761/files#diff-c95083af46065bc635d1536aaca2bc2bc7d9002264822c715391154f6d0ad290R1033-R1052 

but I would like this keep this PR simple and will fix issue#2 and issue#3 later.